controller/settingscontroller.php

```diff
@@ -257,24 +274,24 @@ public static function getFile($userLang, $defaultLang, $basePath, $fileExt,
                     'readable by the apache user';
 
                 //This ensures that carriage returns appear in a textarea
-                $message = Utils::fixCarriageReturns($message); 
-                \OCP\Util::writeLog($appId, $message, \OCP\Util::FATAL);
+                $message = $this->utils->fixCarriageReturns($message);
```

Comment:
Again, static is not needed, just pass in the utils class by adding it to the constructor